bin/apisix

```diff
@@ -1025,7 +1025,7 @@ local function init_etcd(show_output)
         for _, dir_name in ipairs({"/routes", "/upstreams", "/services",
                                    "/plugins", "/consumers", "/node_status",
                                    "/ssl", "/global_rules", "/stream_routes",
-                                   "/proto"}) do
+                                   "/proto", "/plugin_metadata"}) do
```

Comment:
What is the difference between metadata and the properties of plugin itself?  I didn't see why we need it.

Reply:
get plugin schema: 
/apisix/admin/plugins/{plugin-name}, eg: /apisix/admin/plugins/limit-req

get plugin metadata:
/apisix/admin/plugin_metadata/{plugin-name}, eg: /apisix/admin/plugin_metadata/example-plugin

Reply:
https://github.com/apache/apisix/pull/2294#issuecomment-698108183

Reply:
why we need this PR?

Reply:
I would recommend using `/plugins/{name}/metadat` in a RESTful way if this PR is needed..

Reply:
I am also curious about what scenarios need metadata instead of plugin's option.

Reply:
this PR has been merged. we can talk in here: https://github.com/apache/apisix/pull/2307#issuecomment-698337991